wiki/en/Include-Backing-Up.md

```diff
@@ -1,15 +1,15 @@
 <!-- NOTE: This must apply to both Client and Server, and all operating systems -->
 
-Once installed and running, you may want to keep a copy of your settings. Having a backup is always a good idea, and settings files are not backwardly compatible between versions of Jamulus. So if you want to go back to the previous version, you will need to restore the settings you had. 
+Once installed and running, you may want to keep a copy of your settings. Having a backup is always a good idea, and settings files are not backwardly compatible between versions of Jamulus. So if you want to go back to the previous version, you will need to restore the settings you had.
 
-To find your settings file on Windows, look in the `%APPDATA%` folder. There will be one or more `.ini` files in a folder there named `Jamulus`.
+To find your settings file on Windows, launch the file manager and type `%APPDATA%` into the folder path box at the top. After that, search for the a folder named `Jamulus`. There will be one or more `.ini` files in this folder. Now back up Jamulus' settings by copying these files to a different location.
 
-For all other platforms, run the following from the command line to find where they live: 
+For all other platforms, run the following from the command line to find where they live:
```

Comment:
It's probably worth another note after the `find` to back up all Jamulus ini files in the same location, and then a final comment to remind people to check for any `--inifile` parameters they've chosen to specify and back the mentioned files up, too.

Reply:
```suggestion
For all other platforms, run the following from the command line to find where they live and copy the files to another location:
```

Reply:
Yeah, that's better.